plugins/epos/plugin.py

```diff
@@ -277,6 +285,9 @@ def rda_f1_01m(self, **kwargs):
         points, msg_list = self.eval_persistency(id_list, data_or_metadata="metadata")
         logger.debug(msg_list)
 
+        if not self.metadata_persistance:
+            if points == 100:
+                points = 50
```

Comment:
Document this policy in the docstring above

Reply:
Documented

Reply:
I realize now that we should not be that strict. If `eval_persistency()` returns a successful response, then the check should be ok (points = 100). Otherwise, check the `metadata_persistence` and if exists (and resolvable) then points=100 as well (points=0 if not provided). What do you think?

BTW note that it is **persistence**, not persistance